server/core/config/raw/jobs.go

```diff
@@ -0,0 +1,58 @@
+package raw
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+)
+
+/*
+Ref: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html
+S3 bucket naming rules:
+1. Must be between 3 - 63 characters long
+2. Must be lowercase letters, numbers, dots and hyphens.
+*/
+const ValidBucketNameRegEx = `^[a-z0-9-.]*$`
```

Comment:
This can be removed right?

Reply:
yup thanks for the catch! I've removed it 👍